chef/cookbooks/updater/recipes/default.rb

```diff
@@ -17,6 +17,21 @@
 # limitations under the License.
 #
 
+include CrowbarPacemaker::MaintenanceModeHelpers
+
+def set_maintenance_mode
+  if maintenance_mode_set_via_this_chef_run?
+    Chef::Log.info("chef-client run pid #$$ already placed this node in Pacemaker maintenance mode")
+  elsif maintenance_mode?
+    Chef::Log.info("Something else already placed this node in Pacemaker maintenance mode")
+  else
+    execute "crm --wait node maintenance" do
```

Comment:
Hmm, but that only works when run from the corosync nodes; the remote nodes don't even have `crm` installed (and even if they did, I'm not sure if `crm node maintenance` without the extra node name argument would work).